components/Navbar/Navbar.tsx

```diff
@@ -62,9 +73,18 @@ export const Navbar: React.FC<NavbarProps> = ({ isClient }) => {
                 </button>
             </div>
             <div className="Navigation">
-                <NavInstance instanceName={"Root"} />
-                {navInstances}
-                <NavSearchBar placehover={placehover} isMapOpened={isMapOpened} />
+                {/* <NavInstance instanceName={"Root"} /> */}
+                {navInstances.map(instanceData => instanceData.instance)}
+
+                {/* <NavSearchBar
```

Comment:
the commented code is because i hided some peaces of code that are still in progress (not ready for the mvp publishing), for example this case that you commented is a searchBar, the ui is ready, but the logic for searching spaces is not even started.